app/views/review_queues/_responses.html.erb

```diff
@@ -1,10 +1,33 @@
 <div class="panel panel-default">
   <div class="panel-body">
-    <% @queue.responses.each do |r| %>
-      <%= link_to r[0], submit_review_path(name: queue.name, item_id: item.id, response: r[1]), class: 'btn btn-primary review-submit-link',
+    <% if (item.completed and ReviewResult.exists?(item: item)) or
+        ReviewResult.where(user: current_user, item: @item).where.not(result: "skip").exists? %>
+    <h5>
+      <% if item.completed %>
+        Review completed.
+      <% else %>
+        You have already reviewed this item.  It needs more reviews from other users to be completed.
+      <% end %>
+    </h5>
+    <% ReviewResult.where(item: @item).each do |result| %>
+      <b><%= result.user.username %></b>
+      <span
+        data-livestamp="<%= result.created_at.to_i %>"
+        title="<%= result.created_at.to_s %>"
+        > <%= time_ago_in_words(result.created_at.to_s) %>
+      </span>
+      ago: <b><%= result.result.titleize %></b>
+      <br/>
+    <% end %>
+
+    <%= link_to 'Next', 'javascript:void(0)', class: 'btn btn-primary review-next-link' %>
+    <% else %>
+      <% @queue.responses.each do |r| %>
```

Comment:
You should be able to `render 'review_queues/responses', item: item, queue: @queue` here instead of reconstructing all the buttons.

Reply:
*rolls eyes* I'm having a good day today

Reply:
Hooowever, since this is the responses partial, you want `item` and `queue` not `@item` or `@queue`

Reply:
No worries :)

Reply:
Fixed!